### PR TITLE
Release 3.1.0 RC5

### PR DESCRIPTION
**Pull request by KengoTODA**
After merging this PR, a478e9a32df6f45c93bc82b9a87c65c4485fa9c8 (or rebased one) will be tagged as `3.1.0_RC5`.

### CODE REVIEW

File: docs/locale/ja/LC_MESSAGES/installing.po [hunk 1]
```diff
@@ -75,9 +75,9 @@ msgstr ""
 #~ "is to download a binary distribution."
 #~ " Binary distributions are available in "
 #~ "`gzipped tar format "
-#~ "<https://github.com/spotbugs/spotbugs/archive/3.1.0_preview2.tar.gz>`_"
+#~ "<http://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/3.1.0-RC4/spotbugs-3.1.0-RC4.tgz>`_"
```
Comment (iloveeclipse) [inline docs/locale/ja/LC_MESSAGES/installing.po]: Here and line below: really RC4?

Comment (KengoTODA) [inline docs/locale/ja/LC_MESSAGES/installing.po]: This is comment which stores previous text, so I think we can ignore this line.

Comment (iloveeclipse) [inline docs/locale/ja/LC_MESSAGES/installing.po]: A-ha. Interesting and surprising.

**Comment by KengoTODA** [comment]
I've confirmed that Travis CI successfully updates eclipse-candidate update site.

https://github.com/spotbugs/eclipse-candidate/commit/8575903eb63e53b503088b22a723bc0c90168bcd

**Pull request merged**